Jenkinsfile_CNP

```diff
@@ -22,4 +22,8 @@ static LinkedHashMap<String, Object> secret(String secretName, String envVar) {
 
 withPipeline(type, product, app) {
   loadVaultSecrets(secrets)
+  before('buildinfra:aat') {
+    currentBuild.description = "Success, No deployment!"
+    return
```

Comment:
apparently return doesn't work... jui team tried it

Reply:
well it seems to pass on the CI server ... hmm

Reply:
the point is on master you want it to stop, try changing it to something like `build` and see if it stops there